GitUI/CommandsDialogs/FormStash.cs

```diff
@@ -257,14 +257,15 @@ private void FormStashShown(object sender, EventArgs e)
             RefreshAll();
         }
 
-        private void splitContainer2_SplitterMoved(object sender, SplitterEventArgs e)
+        private void splitContainer1_SplitterMoved(object sender, SplitterEventArgs e)
         {
-            Stashes.Size = new Size(Math.Min(200, toolStrip1.Width - 25 - toolStripButton1.Width - toolStripLabel1.Width - toolStripButton_customMessage.Width), Stashes.Size.Height);
+            Stashes.Size = new Size(toolStrip1.Width - 15 - refreshToolStripButton.Width - showToolStripLabel.Width - toolStripButton_customMessage.Width, Stashes.Size.Height);
+            Stashes.DropDownWidth = Math.Max(Stashes.Size.Width, splitContainer1.Width - 2 * showToolStripLabel.Width);
```

Comment:
Please use https://github.com/gitextensions/gitextensions/blob/2d15fc1a68f999d56f08326e759becb35fdbb1f9/GitUI/UserControls/ComboBoxHelper.cs#L8

Reply:
This is ToolStripComboBox, not ComboBox so ResizeComboBoxDropDownWidth cannot be used.